plugins/animate/fire/fire.cpp

```diff
@@ -88,7 +90,46 @@ class FireTransformer : public wf::view_transformer_t
         p.life = 1;
         p.fade = random(0.1, 0.6);
 
-        p.color = {random(0.4, 1), random(0.08, 0.2), random(0.008, 0.018), 1};
+        wf::color_t color_setting = fire_color;
+        bool random_colors = random_fire_color;
```

Comment:
why can't you use `random_fire_color` directly in the if-statement below?

Reply:
It can be moved there, no real reason for doing it this way. I'll change it in a sec